docs/blitz/design/design.rst

```diff
@@ -197,18 +101,33 @@ Example can be seen below.
         device: PE1
         timeout: 10
 
+
+configure_dual
+^^^^^^^^^^^^^^^
+
+The `configure_dual` action is used to configure the device. It is mandatory to commit your config command to end the session
```

Comment:
Nxos supports dual configuration, where a commit is necessary. In these case, use this action.

Reply:
Done